src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.ImplicitInstanceInfo.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.FlowAnalysis
+{
+    internal partial class ControlFlowGraphBuilder
+    {
+        /// <summary>
```

Comment:
Nit: the comment being duplicated makes me think one of the two will go out of date.
Consider phrasing so that one the master (comprehensive) and the other just a summary (with list of examples, but clearly non-comprehensive).